dhis-2/dhis-services/dhis-service-audit-consumer/src/main/java/org/hisp/dhis/audit/legacy/MetadataAuditConsumer.java

```diff
@@ -67,18 +65,16 @@ public MetadataAuditConsumer(
         this.auditService = auditService;
         this.objectMapper = objectMapper;
 
-        this.metadataAuditLog = Objects.equals( dhisConfig.getProperty( ConfigurationKey.METADATA_AUDIT_LOG ), "on" );
+        this.isAuditLogEnabled = Objects.equals( dhisConfig.getProperty( ConfigurationKey.AUDIT_LOGGER ), "on" );
```

Comment:
Move the default value to `ConfigurationKey#defaultValue`. This way we can centralize it.